tests/testsuite/docker.rs

```diff
@@ -0,0 +1,51 @@
+use ansi_term::Color;
```

Comment:
this needs a rebase against current master (where tests are at the ed of the `modules/<module>.rs` file)